homeassistant/components/environment_canada/sensor.py

```diff
@@ -215,7 +215,7 @@ class ECSensorEntityDescription(
     device_class=SensorDeviceClass.AQI,
     native_unit_of_measurement="AQI",
     state_class=SensorStateClass.MEASUREMENT,
-    value_fn=lambda data: data.current,
+    value_fn=lambda data: _get_aqhi_value(data),  # pylint: disable=unnecessary-lambda
```

Comment:
```suggestion
    value_fn=_get_aqhi_value
```
If the parameters match the function parameters you can remove the lambda